### ISSUE TITLE
brave not starting due: cannot change mount namespace of snap

### ISSUE THREAD

**Issue by jangorecki** [brave/browser-laptop#15338]
- [x] Yes I did try the new version
`apt update && apt dist-upgrade` done

- [x] I believe this issue is critical for users (security issue, bug that prevents folks from using the software)
yeah, cannot access my browser

- [x] I've read the `FAQs and Common Issues` section on community.brave.com (https://community.brave.com/c/common-issues)
found [similarly named issue](https://community.brave.com/t/brave-does-not-start-in-ubuntu-18/37574) there but error messages are totally different

After months of very good experience using Brave just now it stopped opening.
To reproduce

  1. clicking brave icon from launcher
  2. or starting brave from shell

The latter produce:
```
main.go:224: cannot change mount namespace of snap "brave" according to change mount (/snap/gtk-common-themes/818/share/icons/Suru /snap/brave/44/data-dir/icons/Suru none bind,ro 0 0): cannot use "/snap/gtk-common-themes/818/share/icons/Suru" as bind-mount source: not a directory
Gtk-Message: Failed to load module "canberra-gtk-module"
Gtk-Message: Failed to load module "canberra-gtk-module"
Gkr-Message: secret service operation failed: An AppArmor policy prevents this sender from sending this message to this recipient; type="method_call", sender=":1.96" (uid=1000 pid=3645 comm="/snap/brave/44/opt/brave.com/brave/brave --no-sand" label="snap.brave.brave (enforce)") interface="org.freedesktop.DBus.Peer" member="Ping" error name="(unset)" requested_reply="0" destination="org.freedesktop.secrets" (uid=1000 pid=2072 comm="/usr/bin/gnome-keyring-daemon --daemonize --login " label="unconfined")
[3645:3906:1215/113150.934071:ERROR:rewards_service_impl.cc(133)] Failed to read file: /home/jan/snap/brave/44/.config/BraveSoftware/Brave-Browser/Default/ledger_state
[3645:4057:1215/113202.916344:ERROR:udev_watcher.cc(60)] Failed to begin udev enumeration.
```

Version 0.56.12 Chromium: 70.0.3538.77 (Official Build) unknown (64-bit)
```
snap info brave
```
```
name:      brave
summary:   A browser with your interests at heart.
publisher: Brave Software (brave*)
contact:   https://community.brave.com/
license:   unset
description: |
  Browse faster by blocking ads and trackers that violate your privacy and
  cost you time and money.
commands:
  - brave
snap-id:      uE3hSmGE91m9MpbDEnUWi2vpeumH6gmv
tracking:     stable
refresh-date: today at 10:26 IST
channels:                              
  stable:    v0.56.12-5.dev (44) 160MB -
  candidate: ^                         
  beta:      ^                         
  edge:      ^                         
installed:   v0.56.12-5.dev (44) 160MB -
```

**Comment by jangorecki** [comment]
looks like snap issue.
similar ones https://bugs.launchpad.net/ubuntu/+source/gnome-calculator/+bug/1794953
should be already fixed in 2.36 (I am running 2.36.2)

**Comment by bsclifton** [comment]
@jangorecki can you please try the snap version for the NEW version of Brave?
https://github.com/brave/brave-browser/issues/1007

Please leave comments/questions in that issue 😄 Thanks!
